db/schema.rb

```diff
@@ -10,23 +10,11 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2020_05_12_001305) do
+ActiveRecord::Schema.define(version: 2020_02_27_214321) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
-  create_table "ahoy_events", force: :cascade do |t|
-    t.string "name"
-    t.jsonb "properties"
-    t.datetime "time"
-    t.bigint "user_id"
-    t.bigint "visit_id"
-    t.index ["name", "time"], name: "index_ahoy_events_on_name_and_time"
-    t.index ["properties"], name: "index_ahoy_events_on_properties", opclass: :jsonb_path_ops, using: :gin
-    t.index ["user_id"], name: "index_ahoy_events_on_user_id"
-    t.index ["visit_id"], name: "index_ahoy_events_on_visit_id"
-  end
-
```

Comment:
removed the table associated with the `ahoy-matey` gem

Reply:
Will we not need this table going forward?